From 405ad723487a4f1bea0e515491d5443df1fe62c9 Mon Sep 17 00:00:00 2001
From: Steven Moreland <smoreland@google.com>
Date: Tue, 18 Jul 2017 16:53:53 -0700
Subject: [PATCH 65/74] libfm_jni: use proper nativehelper headers

libnativeheader exports headers under nativeheader. These were
available before incorrectly as global headers in order to give
access to jni.h.

Bug: 63762847
Change-Id: I79ac33728bee7daada17c05146776aa3a006ae2c
---
 libfm_jni/LibfmJni.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libfm_jni/LibfmJni.cpp b/libfm_jni/LibfmJni.cpp
index 51657da..19baf66 100644
--- a/libfm_jni/LibfmJni.cpp
+++ b/libfm_jni/LibfmJni.cpp
@@ -30,7 +30,7 @@ IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #define LOG_TAG "android_hardware_fm"
 
 #include <jni.h>
-#include "JNIHelp.h"
+#include <nativehelper/JNIHelp.h>
 #include "android_runtime/AndroidRuntime.h"
 #include <utils/Log.h>
 #include "FmRadioController.h"
-- 
2.17.1

